app/scripts/views/components/shareVirtualStudy/shareVirtualStudy.js

```diff
@@ -0,0 +1,206 @@
+/**
+ * Share virtual cohort component
+ * Created by Jing Su on 9/25/2017
+ */
+
+'use strict';
+(function(Vue, $, vcSession) {
+  Vue.component('shareVirtualStudy', {
+    template:
+    '<div v-if="showShareButton" class="share-virtual-study">' +
+    '<div class="share-cohort-btn">' +
+    '<i class="fa fa-share-alt" alt="Share Virtual Study"></i></div></div>',
+    props: [
+      'selectedPatientsNum', 'selectedSamplesNum',
+      'stats', 'updateStats', 'showShareButton'
+    ],
+    data: function() {
+      return {
+        savedVC: null
+      };
+    }, methods: {
+      saveCohort: function() {
+        var _self = this;
+        _self.updateStats = true;
+        _self.$nextTick(function() {
+          _self.addNewVC = true;
+        });
+      }
+    },
+    ready: function() {
+      var self_ = this;
+      var previousSelectedCases = {};
+      if (this.showShareButton) {
+        $('.share-virtual-study').qtip({
```

Comment:
Do you think we can wrap up the qtip configuration to util? I think most of qtips are sharing the same configuration, only the content is different.

Reply:
I'm not sure since html code of saveButton and shareButton are different.

Reply:
The qTio configuration code is almost identical, we can probably abstract that part.